src/main.py

```diff
@@ -1,16 +1,28 @@
-import argparse, os, json, time
+import argparse
+import os
+import json
+import time
+from typing import List, Tuple
 from tqdm import tqdm
-import numpy as np, pandas as pd
+import numpy as np
```

Comment:
@3ripleM the general idea is not to add vertical lines :)

Reply:
This code style makes the codes harder to read. it is not bad to write a lot of code. it is bad that the code is not readable.
I know that python developers can understand that but they need to focus on understanding what is imported from where and also what is the aliases. 

This is a report on UX development and why long lines are bad for readability.
https://baymard.com/blog/line-length-readability

Reply:
@3ripleM 
you're right. but for trivial packages that are imported in almost all ml project like numpy and the likes, all in this domain have reached to a standard alias like np or pd ... 

however, for those that are new or are from our own architecture, we follow what you suggested.